player_rating/tests.py

```diff
@@ -66,11 +66,11 @@ def test_create_player_rating_invalid_rating2(self, player_rating):
                 rating=0)
 
     def test_delete_player_deletes_player_ratings(self, player, player_rating):
-        assert Player.objects.count() == 1
-        assert PlayerRating.objects.count() == 1
+        all_players_ratings_before = PlayerRating.objects.count()
+        player_fixture_ratings = PlayerRating.objects.filter(player=player).count()
```

Comment:
What if this one returns zero? If we assert that `player_fixture_ratings` is greater than zero then we can address this issue and also make the line below redundant. i.e. you could remove this code `assert Player.objects.filter(user=player.user).exists()`.

Reply:
@yftacherzog how it can return zero? isn't it a fixture that created for the test? `test_get_player_rating` is checking if the fixture created. what am I missing?

Reply:
I think the code `assert Player.objects.filter(user=player.user).exists()` is redundant even without asserting `player_fixture_ratings` is greater than zero no?

Reply:
> I think the code `assert Player.objects.filter(user=player.user).exists()` is redundant even without asserting `player_fixture_ratings` is greater than zero no?

Yes. I think it's redundant anyway.


> @yftacherzog how it can return zero? isn't it a fixture that created for the test? `test_get_player_rating` is checking if the fixture created. what am I missing?

The fixture is making it so that it won't be zero. Adding that line can make the test more self explanatory. By adding that single line you are making it more clear what you're testing there without having to go elsewhere to figure out what the fixture is about. 

Having said that, it's not a must to have it in this case. It's a matter of preference. I'll let you decide.

Reply:
Ok thanks. I changed the test to include suggested assertion.